crates/wash-lib/src/build/component.rs

```diff
@@ -710,6 +723,8 @@ func main() {}
                     name: "test".into(),
                     version: Version::parse("0.1.0")?,
                     revision: 0,
+                    wit_path: project_dir.path().join("wit"),
```

Comment:
OK, so should we actually call this `wit_dir` and `build_dir` ?

We really are expecting these folders to be directories I think